test/Chirp.Infrastructure.Tests/CheepRepositoryTests.cs

```diff
@@ -54,65 +54,177 @@ public async void CanCreateCheep(string author, string text, string timeStamp)
 
         // Act
         _repository.CreateCheep(cheepDTO);
-
-        // Assert
         IEnumerable<CheepDTO> cheeps = await _repository.GetCheepsFromAuthorAsync(author);
         CheepDTO cheep = cheeps.Single();
 
+        // Assert
         Assert.Single(cheeps);
         Assert.Equal(author, cheep.Author);
         Assert.Equal(text, cheep.Text);
         Assert.Equal(timeStamp, cheep.TimeStamp);
         Assert.Equal(cheepDTO, cheep);
     }
 
+    [Fact]
+    public void CanCreateCheepWithLongText()
+    {
```

Comment:
Really great to test this! I'm wondering if it would make more sense to change the method name to CantCreateCheepWithLongText would be more representative?